tithemi/assignment3/Test.cpp

```diff
@@ -0,0 +1,70 @@
+#include "gtest/gtest.h"
+#include "Dictionary.hpp"
+#include "LettersGrid.hpp"
+
+static const std::vector<std::string> words =
+{
+    "CAR", "CARD", "CART", "CAT"
+};
+
+static const std::vector<std::string> prefixes =
+{
+    "C", "CA", "CAR", "CARD", "CART", "CAT"
+};
+
+static const int rows = 2;
+static const int columns = 3;
+
+static const std::array<std::array<char, columns>, rows> letters = { {
+    { 'A', 'A', 'R' },
+    { 'T', 'C', 'D' }
+} };
+
+TEST(DictionaryIsWorking, Creates)
+{
+    Dictionary dictionary(words);
+}
+
+TEST(DictionaryIsWorking, ContainsWords)

```

Comment:
Ah, you already test the contents of the dictionary. In that case I would just delete the first test case.